aqua/constants.aqua

```diff
@@ -0,0 +1,3 @@
+const REPLICATION_FACTOR = 1
+const CONSISTENCY_LEVEL = 1
+const DEFAULT_TIMEOUT = 6000
```

Comment:
Why these values? Let's have replication to `5`, and consistency level to `3`

Also, default timeout everywhere is 7000, maybe keep it that way? Couldn't give a reason for that, though. Besides consistency, that is.

Reply:
I used minimum values in the first place because the overall execution was slow. But now it's okay to have bigger values.

The default timeout for a particle is 7000, and I thought that timeout for gathering results should be smaller